service/src/main/java/uk/gov/hmcts/reform/fpl/model/Other.java

```diff
@@ -158,4 +158,19 @@ public boolean isDeceasedOrNFA() {
         return AddressNotKnowReason.DECEASED.getType().equals(addressNotKnowReason)
             || AddressNotKnowReason.NO_FIXED_ABODE.getType().equals(addressNotKnowReason);
     }
+
+    public Other removeAddress() {
+        Other other =  this.toBuilder()
+            .address(null)
+            .build();
+        return other;
+    }
+
+    public Other removeAddressNotKnowReason() {
+        Other other =  this.toBuilder()
```

Comment:
Same here as above, just return the builder call. No need to create a variable then return the variable

Reply:
changed